src/deals/list/deals.html

```diff
@@ -13,6 +13,12 @@
         click.delegate="ethereumService.ensureConnected()">
         Connect to Show Your Deals
       </pbutton>
+      <pbutton
```

Comment:
It would be ideal, however then we cannot show "connect to show your deals" button if the wallet is already connected. We also cannot show "Show my deals only" toggle because they are not authenticated. 

So we can show nothing, or have third scenario, and show "authenticate to show your deals" button. It was simple to hook it up to the dealService and seems to work well.